platform/jobs/pipelines/edxPlatformPipelinePr.groovy

```diff
@@ -271,6 +274,7 @@ jobConfigs.each { jobConfig ->
             environmentVariables(
                 REPO_NAME: "${jobConfig.repoName}",
                 TOX_ENV: "${jobConfig.toxEnv}"
+                PYTHON_VERSION: "${jobConfig.pythonVersion}"
```

Comment:
This doesn't trigger an error for configs in which `pythonVersion` wasn't defined?

Reply:
No. We follow the same pattern for setting `TOX_ENV`. Groovy just evaluates it to null (which I check for in the platform script).